src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/X509Certificates/X509CertificateLoader.Android.cs

```diff
@@ -52,24 +53,43 @@ private static partial Pkcs12Return FromCertAndKey(CertAndKey certAndKey, Import
         {
             AndroidCertificatePal pal = (AndroidCertificatePal)certAndKey.Cert!;
 
-            if (certAndKey.Key != null)
+            if (certAndKey.Key is Pkcs12Key pkcs12Key)
```

Comment:
Since the type of CertAndKey.Key is Pkcs12Key, this is just a null check.  But it looks like type testing.

I'd prefer that we go back to just a null check and s/pkcs12Key/certAndKey.Key/g